be/src/connector/hive_connector.cpp

```diff
@@ -71,6 +71,7 @@ Status HiveDataSource::open(RuntimeState* state) {
         _no_data = true;
         return Status::OK();
     }
+    SCOPED_TIMER(_profile.scan_timer);
```

Comment:
I'm not sure it's necessary to include scanner initial time to scan time

Reply:
There is no harm to time it.